code/game/objects/structures/morgue.dm

```diff
@@ -144,20 +144,20 @@
 	qdel(connected)
 
 /obj/structure/morgue/attackby(obj/item/P, mob/user)
-	if(iscrowbar(P)&&!contents.len)
+	if(iscrowbar(P) && !contents.len)
 		user.visible_message("<span class='notice'>\The [user] begins dismantling \the [src].</span>", "<span class='notice'>You begin dismantling \the [src].</span>")
-		if(do_after(user, src,50))
+		if(do_after(user, src, 50) && !contents.len)
 			user.visible_message("<span class='notice'>\The [user] dismantles \the [src].</span>", "<span class='notice'>You dismantle \the [src].</span>")
 			playsound(src, 'sound/items/Deconstruct.ogg', 50, 1)
 			new /obj/structure/closet/body_bag(src.loc)
-			new /obj/item/stack/sheet/metal(src.loc,5)
+			new /obj/item/stack/sheet/metal(src.loc, 5)
 			qdel(src)
 	if(P.is_wrench(user))
 		P.playtoolsound(src, 50)
-		if(dir==4)
-			dir=8
+		if(dir == 4)
+			dir = 8
```

Comment:
are we allowed to change format things like this in prs?? i hope so cuz i hate this style

Reply:
yeah, i find it being spread out like that is much easier to read
shame it kinda clutters the diff tho

Reply:
This kind of thing is fine